pinery-lims/src/main/java/ca/on/oicr/pinery/lims/SampleAttribute.java

```diff
@@ -29,7 +29,7 @@
     SAMPLE_RECEIVE_DATE("geo_receive_date", "Receive Date"),
     EXTERNAL_NAME("geo_external_name", "External Name"),
     TUBE_ID("geo_tube_id", "Tube ID"),
-    TARGETED_RESEQUENCING("geo_targeted_resequencing", "Targeted Resequencing"),
+    TARGETED_RESEQUENCING("geo_targeted_resequencing", "Targeted Resequencing", "Targeted Sequencing"),
```

Comment:
For consistency, Pinery-MISO uses the same attribute names as Pinery-GSLE. Any case where this is not true is a mistake in Pinery-MISO. 'Targeted Sequencing' was already remapped to 'Targeted Resequencing'